src/test/java/uk/gov/register/presentation/functional/TestEntry.java

```diff
@@ -7,9 +7,17 @@
     public final String itemJson;
     public final Instant entryTimestamp;
 
-    public TestEntry(int entryNumber, String itemJson, Instant entryTimestamp) {
+    private TestEntry(int entryNumber, String itemJson, Instant entryTimestamp) {
         this.entryNumber = entryNumber;
         this.itemJson = itemJson;
         this.entryTimestamp = entryTimestamp;
     }
+
+    public static TestEntry anEntry(int entryNumber, String itemJson){
+        return new TestEntry(entryNumber, itemJson, Instant.now());
```

Comment:
should this be `now()` or could it be a fixed value instead? I'm just thinking at a high level about how we can avoid potential sources of test flakiness.